engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -738,7 +738,7 @@ void printUsage(String cmd)
         "Kill a container"));
     connectedCommands.put("shutdown-app", new CommandSpec(new ShutdownAppCommand(),
         null,
-        new Arg[]{new VarArg("app-id")},
+        new Arg[]{new VarArg("app-id/app-name")},
```

Comment:
Similar change is needed at
     globalCommands.put("shutdown-app", new CommandSpec(new ShutdownAppCommand(),
         new Arg[]{new Arg("app-id")},
         new Arg[]{new VarArg("app-id")},

line number 634